kolibri/auth/models.py

```diff
@@ -11,7 +11,8 @@
 """
 from __future__ import absolute_import, print_function, unicode_literals
 
-from django.contrib.auth.models import AbstractBaseUser
+from django.contrib.auth.models import AbstractBaseUser, _user_get_all_permissions, _user_has_module_perms, \
```

Comment:
There's something odd about importing a function prefixed `_function_name` because the `_` prefix indicates that its local to its module..